<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CSCSearch</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>16</y>
     <width>781</width>
     <height>571</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1366</width>
     <height>768</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>33</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_search">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>761</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(0, 0,128);
    color: rgb(255,255,255);
	font-size: 11px;
    font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_server">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>107.113.53.30:1716</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Branch</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>761</width>
      <height>331</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>759</width>
       <height>329</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="te_message">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>741</width>
        <height>311</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="le_branch">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>631</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>//CSC_NILE/Strawberry/EXYNOS5/a5xelte_NN/EUR/OLB/</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_user">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lam.bich</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_password">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>perforce</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>33</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tag name (without &lt;&gt;)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_go">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(0, 0,128);
    color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sale code</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_sale">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Workspace</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tags values (separed by ,)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_options">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>99</y>
      <width>761</width>
      <height>81</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    background-color: rgb(178, 225, 255);
	font-size: 11px;
    font-weight: bold;
}
QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0px;
    padding-left: 5px;
    color:blue
}</string>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QLineEdit" name="le_tag_values">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>601</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>23410,23415</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Version: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_tag_name">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MCCMNC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_author">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>550</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> lam.bich</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_wsp">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <zorder>gb_options</zorder>
   <zorder>label</zorder>
   <zorder>pb_search</zorder>
   <zorder>le_server</zorder>
   <zorder>label_3</zorder>
   <zorder>scrollArea</zorder>
   <zorder>le_branch</zorder>
   <zorder>le_user</zorder>
   <zorder>le_password</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>pb_go</zorder>
   <zorder>label_9</zorder>
   <zorder>cb_sale</zorder>
   <zorder>label_4</zorder>
   <zorder>label_10</zorder>
   <zorder>label_version</zorder>
   <zorder>le_tag_name</zorder>
   <zorder>label_author</zorder>
   <zorder>cb_wsp</zorder>
  </widget>
  <widget class="QGroupBox" name="gb_input">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>761</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    background-color:rgb(211, 211, 211);
	font-size: 11px;
    font-weight: bold;
}
QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0px;
    padding-left: 5px;
    color:blue
}</string>
   </property>
   <property name="title">
    <string>Connection</string>
   </property>
  </widget>
  <zorder>gb_input</zorder>
  <zorder>centralwidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
